features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -1,1192 +0,0 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is strange to have as a diff, did you rebase properly?

Reply:
Hm.... I think what I did is I checked if the file exists in master branch of main repository and when I didn't find it, I deleted it. Also I believe the latest push is not rebased to current master. Maybe I should pull master and rebase again...

Reply:
Nevermind git was able to sort it out :-)

Thanks for your hard work!